modules/sandboxed-containers-preparing-openshift-cluster.adoc

```diff
@@ -3,5 +3,105 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-preparing-openshift-cluster_{context}"]
+= Preparing your cluster for {sandboxed-containers-first}
 
-= Preparing your cluster for OpenShift sandboxed containers
+Before you install {sandboxed-containers-first}, ensure that your {product-title} cluster meets the following requirements:
+
+* Your cluster must be installed on bare metal infrastructure with {op-system-first} workers.
```

Comment:
Should you combine these two prereqs so that folks don't think after the first bullet that they can use sandboxes with baremetal UPI?